app/views/govuk_publishing_components/components/_previous_and_next_navigation.html.erb

```diff
@@ -32,7 +32,7 @@
             data-ga4-link = "<%= {
               event_name: "navigation",
               type: "content",
-              text: previous_page[:label],
+              text: previous_page[:label] || title,
```

Comment:
Ah great, ignore my (previous but one?) comment.

Reply:
Saw this after I replied to the other comment!